invenio_cli/helpers/commands.py

```diff
@@ -70,19 +70,11 @@ def _update_instance_path(self):
         ).stdout.strip()
         self.cli_config.update_instance_path(path)
 
-    def _symlink_project_config(self):
-        project_config = 'invenio.cfg'
-        click.secho("Symlinking {}...".format(project_config), fg="green")
-        target_path = self.cli_config.get_project_dir() / project_config
-        link_path = self.cli_config.get_instance_path() / project_config
-        filesystem.force_symlink(target_path, link_path)
-
-    def _symlink_templates(self):
-        """Symlink the templates folder."""
-        templates = 'templates'
-        click.secho('Symlinking {}/...'.format(templates), fg='green')
-        target_path = self.cli_config.get_project_dir() / templates
-        link_path = self.cli_config.get_instance_path() / templates
+    def _symlink_project_file_or_folder(self, target):
```

Comment:
Major: This change is nice! However, it requires more changes (see travis) because it is breaking the local installation:

```
 commands._symlink_project_config()
       AttributeError: 'LocalCommands' object has no attribute '_symlink_project_config'
```

Otherwise it LGTM!

Reply:
Ah sorry, doing this at the end of the day... Should be fixed now.